### DISCUSSION TITLE
RunAsSystem.exe

### DISCUSSION THREAD

**Discussion by jravazzano** [OpenSourceCommunityBrasil/Aegys-Acesso-Remoto#70] [General]
Hola,
Tengo una pregunta. Cuando intento ejecutar el programa me da el error:
  [BRCC32 Error] AegysSuporteClienteResource.rc(1): file not found: Assets\Resources\RunAsSystem.exe 

Buscando en internet encontre en  https://sourceforge.net/projects/runassystem/  el ejecutable RunAsSystem.exe. No se si este ejecutable es el de la referencia en la carpeta de Ressources y si es seguro usarlo en Ressources. Segun lo que he leido es para elevar al maximo los privilegios en Windows. Yo uso el Windows 11 Pro. No se si sera compatible.

Muchas gracias por sus comentarios y/o sugestiones.

Obrigado. 

Julio

**Comment by chipcrazy** [comment]
Hola,
yo tengo el mismo problema, nose si ya lograste hacer funciona ? y como ?

saludos,
Mauricio
